appservice/multiplexer.py

```diff
@@ -44,7 +44,7 @@
 
         {routes}
 
-        location /api/webconsole/v1/sessions/new {{
+        location /api/webconsole {{
```

Comment:
This is generally a good generalization, but I think this should only cover the /api/webconsole/ path. We need to handle the /webconsole-{http,ws} parts completely differently. We use /wss/ for these mostly, but we might still need to go back to /api. So TL/DR: Append a `/` here.